extension/package.json

```diff
@@ -615,8 +615,8 @@
           "type": "boolean",
           "default": null
         },
-        "dvc.doNotShowWalkthroughAfterInstall": {
```

Comment:
If any user has this setting set to true, I believe it will get undone with the name change. We could possibly check for this setting and set the new one to true for them, but probably not worth the effort with the amount of users using it.